app/pages/system/UtilizationPage.tsx

```diff
@@ -254,7 +254,8 @@ const AvailableCell = ({
   return (
     <div className="flex w-full items-center justify-between">
       <div>
-        {allocated - provisioned} {unit && <span className="text-tertiary">{unit}</span>}
+        {round(allocated - provisioned, 2)}
+        {unit && <span className="text-tertiary"> {unit}</span>}
```

Comment:
ah so we did forget to round! the mistake was assuming the difference of two rounded numbers would remain rounded. JAVASCRIPT